java/client/src/main/java/glide/api/BaseClient.java

```diff
@@ -93,34 +93,40 @@ protected static CommandManager buildCommandManager(ChannelHandler channelHandle
     }
 
     /**
-     * Extracts the response from the Protobuf response and either throws an exception or returns the
-     * appropriate response as an <code>Object</code>.
+     * Extracts the value from a Redis response message and either throws an exception or returns the
+     * value as an object of type {@link T}. If <code>isNullable</code>, than also returns <code>null
+     * </code>.
      *
      * @param response Redis protobuf message
-     * @return Response <code>Object</code>
+     * @param classType Parameter {@link T} class type
+     * @param isNullable Accepts null values in the protobuf message
+     * @return Response as an object of type {@link T} or <code>null</code>
+     * @param <T> return type
+     * @throws RedisException on a type mismatch
      */
+    private <T> T handleRedisResponse(Class classType, boolean isNullable, Response response) {
```

Comment:
```suggestion
    protected <T> T handleRedisResponse(Class<T> classType, boolean isNullable, Response response) throws RedisException {
```

Reply:
Thanks @shachlanAmazon for the idea

Reply:
updated in 2cbf558e0900e94258bfc8c51cdc05a4187cd420